scripts/bootstrap-aio.yml

```diff
@@ -132,26 +132,12 @@
       with_items:
         - name: "user_osa_variables_defaults.yml"
           mode: "0440"
-        - name: "env.d/cinder.yml"
         - name: "env.d/elasticsearch.yml"
           condition: "{{ rpco_deploy_elk | bool }}"
-        - name: "env.d/glance.yml"
-        - name: "env.d/heat.yml"
-        - name: "env.d/horizon.yml"
-        - name: "env.d/ironic.yml"
-        - name: "env.d/keystone.yml"
         - name: "env.d/kibana.yml"
```

Comment:
I think that we can move the variant setting for the ELK roles from the env.d files into group_vars too, for consistency.